curriculum/challenges/english/25-front-end-development/lab-availability-table/66b36358ed4f261d64840c24.md

```diff
@@ -121,126 +121,142 @@ You should define a `--color0` variable in your root selector.
 
 ```js
 const root = new __helpers.CSSHelp(document).getStyle(':root');
-assert(!!root && root.getPropVal('--color0'));
+assert.exists(root); 
+assert.exists(root.getPropVal('--color0'))
```

Comment:
```suggestion
assert.isNotEmpty(root.getPropVal('--color0'))
```
Once the root selector exists, the following assert is always satisfied. I thought it wouldn't because this is a custom property but the test passes without creating `--color0`.